custom_components/powercalc/translations/de.json

```diff
@@ -0,0 +1,176 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Sensor ist bereits konfiguriert, geben Sie eine unique_id an"
+        },
+        "error": {
+            "daily_energy_mandatory": "Sie m�ssen mindestens Value oder Valuetemplate angeben",
+            "fixed_mandatory": "Sie m�ssen mindestens Power, Power template oder States power angeben",
+            "fixed_states_power_only": "Diese Entit�t kann nur mit states_power und nicht mit power arbeiten",
+            "group_mandatory": "Sie m�ssen mindestens Untergruppen oder Power- und Energie-Entit�ten definieren",
+            "linear_mandatory": "Sie m�ssen mindestens max_power oder Kalibrierungswert angeben",
+            "linear_min_higher_as_max": "max_power kann nicht niedriger sein als min_power"
+        },
+        "step": {
+            "daily_energy": {
+                "data": {
+                    "name": "Name",
+                    "on_time": "Dauer",
+                    "start_time": "Startzeit",
+                    "unique_id": "Unique id",
+                    "unit_of_measurement": "Ma�einheit",
+                    "update_frequency": "Aktualisierungsfrequenz",
+                    "value": "Value",

```

Comment:
I think this one can also be translated. I don't know the right word for it in german. This is the value for daily energy sensor, could be in W of kWh.

Reply:
It would be "Energieverbauch".

Reply:
I have chosen to not call it energy usage in English because that would only apply to the kWh value. Energy (usage) is the energy used over timespan (in kWh)
The W value (which is in essence the power at a certain moment in time) is called power in english and not energy.

But long story short, I'm fine with Energieverbauch, as there is also more explanation below the input box.

Reply:
> could be in W of kWh.

Ah. not Wh but W in this case? Then I got it wrong.

W is Leistung
Wh is Energie/Energieverbrauch

Reply:
> > could be in W of kWh.
> 
> Ah. not Wh but W in this case? Then I got it wrong.

Yes it is W. Imagine you measured a network switch at 10W. Than you can specify `value: 10` and `unit_of_measurement: W` and the daily fixed energy sensor will calculate the kWh it needs to add per day. So the user does not need to do this calculation.

> 
> W is Leistung Wh is Energie/Energieverbrauch
> 
> Then it would be Leistung here as well.

The value can be either kWh or W

So maybe "Energie/Leistung" would be the best ;-)

Reply:
> Yes it is W. Imagine you measured a network switch at 10W. Than you can specify `value: 10` and `unit_of_measurement: W` and the daily fixed energy sensor will calculate the kWh it needs to add per day. So the user does not need to do this calculation.

I know. 😉 I only got it wrong here in the reference, because you have written kWh above. 😉

Reply:
Again: Aaah, this was in the daily_fixed section and both are possible, I should double-check the context. Hm,  placed the hint in unit_of_measurement now.